feat(icons): added `drone` icon

shopped opened the pull request:
<!-- Insert `closes #issueNumber` here if merging this PR will resolve an existing issue -->
## Description
Adding new drone icon closes #2154 

### Icon use case <!-- ONLY for new icons, remove this part if not icon PR -->
Interfaces that connect with drones
Filters for drone footage
Map layers to show drone locations

### Concept <!-- ONLY for new icons -->
<!-- All of these requirements must be fulfilled. -->
- [X] I have provided valid use cases for each icon.
- [X] I have not added any a brand or logo icon.
- [X] I have not used any hate symbols.
- [X] I have not included any religious or political imagery.

### Author, credits & license<!-- ONLY for new icons. -->
<!-- Please choose one of the following, and put an "x" next to it. -->
- [ ] The icons are solely my own creation.
- [X] The icons were originally created in #2154 by @bernatfortet
- [ ] I've based them on the following Lucide icons: <!-- provide the list of icons -->
- [ ] I've based them on the following design: <!-- provide source URL and license permitting use -->

### Naming <!-- ONLY for new icons -->
<!-- All of these requirements must be fulfilled. -->
- [X] I've read and followed the [naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions)
- [X] I've named icons by what they are rather than their use case.
- [X] I've provided meta JSON files in `icons/[iconName].json`.

### Design <!-- ONLY for new icons -->
<!-- All of these requirements must be fulfilled. -->
- [X] I've read and followed the [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide)
- [ ] I've made sure that the icons look sharp on low DPI displays.
- [ ] I've made sure that the icons look consistent with the icon set in size, optical volume and density.
- [X] I've made sure that the icons are visually centered.
- [X] I've correctly optimized all icons to three points of precision.

## Before Submitting <!-- For every PR! -->
<!-- All of these requirements must be fulfilled. -->
- [X] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [X] I've checked if there was an existing PR that solves the same issue.

jguddas commented:
<a title="Open lucide studio" target="_blank" href="https://studio.lucide.dev/edit?value=%3Cpath+d=%22M10+10+7+7%22+/%3E%3Cpath+d=%22m10+14-2.993+2.994%22+/%3E%3Cpath+d=%22m14+10+3-2.999%22+/%3E%3Cpath+d=%22m14+14+3.004+3.004%22+/%3E%3Cpath+d=%22M14.205+4.139a4+4+0+1+1+5.439+5.863%22+/%3E%3Cpath+d=%22M19.637+13.993a4+4+0+1+1-5.432+5.868%22+/%3E%3Cpath+d=%22M4.367+10.01a4+4+0+1+1+5.438-5.862%22+/%3E%3Cpath+d=%22M9.795+19.862a4+4+0+1+1-5.429-5.873%22+/%3E%3Crect+x=%2210%22+y=%228%22+width=%224%22+height=%228%22+rx=%221%22+/%3E"><img alt="icons" width="200px" src="https://lucide.dev/api/gh-icon/PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNMTAgMTAgNyA3IiAvPgogIDxwYXRoIGQ9Im0xMCAxNC0yLjk5MyAyLjk5NCIgLz4KICA8cGF0aCBkPSJtMTQgMTAgMy0yLjk5OSIgLz4KICA8cGF0aCBkPSJtMTQgMTQgMy4wMDQgMy4wMDQiIC8+CiAgPHBhdGggZD0iTTE0LjIwNSA0LjEzOWE0IDQgMCAxIDEgNS40MzkgNS44NjMiIC8+CiAgPHBhdGggZD0iTTE5LjYzNyAxMy45OTNhNCA0IDAgMSAxLTUuNDMyIDUuODY4IiAvPgogIDxwYXRoIGQ9Ik00LjM2NyAxMC4wMWE0IDQgMCAxIDEgNS40MzgtNS44NjIiIC8+CiAgPHBhdGggZD0iTTkuNzk1IDE5Ljg2MmE0IDQgMCAxIDEtNS40MjktNS44NzMiIC8+CiAgPHJlY3QgeD0iMTAiIHk9IjgiIHdpZHRoPSI0IiBoZWlnaHQ9IjgiIHJ4PSIxIiAvPgo8L3N2Zz4=.svg"/><br/>Open lucide studio</a>

shopped commented:
> [<img alt="icons" width="200px" src="https://camo.githubusercontent.com/938ac5dc01ba0fe543085e574c74d8b530bc33fdb13be3832e1645a50048251d/68747470733a2f2f6c75636964652e6465762f6170692f67682d69636f6e2f50484e325a776f67494868746247357a50534a6f644852774f693876643364334c6e637a4c6d39795a7938794d4441774c334e325a79494b4943423361575230614430694d6a516943694167614756705a326830505349794e43494b4943423261575633516d393450534977494441674d6a51674d6a5169436941675a6d6c7362443069626d39755a53494b4943427a64484a7661325539496d4e31636e4a6c626e5244623278766369494b4943427a64484a766132557464326c6b64476739496a496943694167633352796232746c4c577870626d566a59584139496e4a766457356b49676f6749484e30636d39725a5331736157356c616d3970626a3069636d3931626d5169436a344b49434138634746306143426b50534a4e4d5441674d5441674e7941334969417650676f67494478775958526f49475139496d30784d4341784e43307a49444d694943382b4369416750484268644767675a443069625445304944457749444d744d7949674c7a344b49434138634746306143426b50534a744d5451674d5451674d79417a4969417650676f67494478775958526f49475139496b30784e4334794d4455674e4334784d7a6c684e434130494441674d534178494455754e444d35494455754f44597a4969417650676f67494478775958526f49475139496b30784f5334324d7a63674d5452684e434130494441674d5341784c5455754e444d79494455754f4459344969417650676f67494478775958526f49475139496b30304c6a4d324e7941784d474530494451674d434178494445674e5334304d7a67744e5334344e6a49694943382b4369416750484268644767675a44306954546b754e7a6b31494445354c6a67324d6d4530494451674d434178494445744e5334304d6a6b744e5334344e7a4d694943382b4369416750484a6c59335167654430694d54416949486b39496a6769494864705a48526f505349304969426f5a576c6e61485139496a676949484a34505349784969417650676f384c334e325a7a343d2e737667">Open lucide studio](https://studio.lucide.dev/edit?value=%3Cpath+d=%22M10+10+7+7%22+/%3E%3Cpath+d=%22m10+14-3+3%22+/%3E%3Cpath+d=%22m14+10+3-3%22+/%3E%3Cpath+d=%22m14+14+3+3%22+/%3E%3Cpath+d=%22M14.205+4.139a4+4+0+1+1+5.439+5.863%22+/%3E%3Cpath+d=%22M19.637+14a4+4+0+1+1-5.432+5.868%22+/%3E%3Cpath+d=%22M4.367+10a4+4+0+1+1+5.438-5.862%22+/%3E%3Cpath+d=%22M9.795+19.862a4+4+0+1+1-5.429-5.873%22+/%3E%3Crect+x=%2210%22+y=%228%22+width=%224%22+height=%228%22+rx=%221%22+/%3E)

I like how this looks! Re-pushed the code with your design, added your name to contributors, and also rebased on main. The vercel deploy failed and I'm not sure why though, maybe you can help or I can try re-opening?
